pkg/images/enricher/enricher_impl_test.go

```diff
@@ -486,7 +486,7 @@ func TestEnricherFlow(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, c.result, result)
 
-			assert.Equal(t, c.fsr, fsr)
+			assert.Equal(t, c.fsr, fsr) //nolint:donotcompareproto false possitive
```

Comment:
What should be compared here ? Can the scanner team answer ? Can the assertion be split (like should only the configuration fields of the wrapper object be compared) ?

Reply:
@stackrox/scanner Could you take a look?

Reply:
Finaly I've changed the type inside fsr struct to a struct not an interface so there is no false positive